react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -79,6 +79,9 @@
     private CountDownLatch mGetContentCountDownLatch;
     private WeakReference<View> mLastFocusedView = null;
 
+    public static final String MEDIA_SOURCE_STOCK_MEDIA = "MEDIA_SOURCE_STOCK_MEDIA";
+    public static final String MEDIA_SOURCE_GIPHY_MEDIA = "MEDIA_SOURCE_GIPHY_MEDIA";
```

Comment:
I wouldn't mention "stock media" or "giphy" at the bridge level. I would say that they are all "extra options".
Ideally we would be able to add more media sources on the client app (like the iOS-only `Other apps` option) without the need to modify the Bridge.

Reply:
Hey @etoledom that was my initial solution but I had a problem with this part of the code: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1469/files#diff-37882e44592184b021430fb0aae5d21cR129-R135

For media stock, we don't have information about the upload process
which means that we need to listen for `MediaSelectedCallback` ( and not  `MediaUploadCallback` )
and for Giphy we have information about the upload process so we need to use `MediaUploadCallback`

Some ugly solution would be to put prefix `upload` in the key so that we know which callback we need to assign to the media source.

Do you have something on your mind in this case?

Reply:
I see, that makes sense.

Just an idea: Do you think it would be possible to use a single progress-based callback (`MediaUploadCallback`?), and send the progress as completed from the beginning in the Stock Photos case?

This might be a simpler workaround 🤔

Reply:
Hey @etoledom I managed to only use one callback for all cases (`MediaUploadCallback`)
which gives us the opportunity to hide 3rd party options from RN layer.

Let me know when you do another iteration of review. Thanks.